### DISCUSSION TITLE
求助，机器人突然报错。

### DISCUSSION THREAD

**Discussion by Ning-Off** [nonebot/discussions#309] [问答]
![image](https://user-images.githubusercontent.com/102615856/216042907-e465fc94-4831-4d4e-b7c1-8b292cf2641b.png)

**Comment by Ning-Off** [comment]
![image](https://user-images.githubusercontent.com/102615856/216044872-f497df2a-740c-4917-b2a5-755550d52ff4.png)
重装依赖后再次报错

**Reply by yanyongyu** [reply]
版本问题吧
